tests/vmcheck/overlay.sh

```diff
@@ -59,12 +59,16 @@ fi
 cd /ostree/repo/tmp
 rm vmcheck -rf
 ostree checkout $commit vmcheck --fsync=0
+# https://github.com/projectatomic/rpm-ostree/pull/693
+rm -f vmcheck/usr/etc/{.pwd.lock,passwd-,group-,shadow-,gshadow-,subuid-,subgid-}
```

Comment:
Can we make the comment there a bit more explicit re. the fact that this should be temporary? (Something like the nice ✂️ ✂️ ✂️ below).